plugin/src/main/scala/org/flywaydb/play/ConfigReader.scala

```diff
@@ -32,49 +35,65 @@ class ConfigReader(configuration: Configuration, environment: Environment) {
   def getFlywayConfigurations: Map[String, FlywayConfiguration] = {
     (for {
       dbName <- getAllDatabaseNames
+      // Will only iterate if this is a configuration group, instead of a single value, allowing easy skipping of irrelevant entries
+      subConfig <- catching(classOf[PlayException]).opt(configuration.getConfig("db." + dbName)).flatten
+
+      // Skip silently if we've been told to
+      if !subConfig.getBoolean("flywayIgnore").getOrElse(false)
+
+      urlOption = subConfig.getString("url")
+      driverOption = subConfig.getString("driver")
+
```

Comment:
This was based on the previous code:

```scala
  val (url, parsedUser, parsedPass) = configuration.getString(s"db.${dbName}.url").map(urlParser.parseUrl(_)).getOrElse(
	throw new MigrationConfigurationException(s"db.${dbName}.url is not set.")
  )
  val driver = configuration.getString(s"db.${dbName}.driver").getOrElse(
	throw new MigrationConfigurationException(s"db.${dbName}.driver is not set.")
  )
```

The separate checks were maintained to keep the clarity of the two different messages, though they were converted to warnings.

Additionally, I considered a complete and valid configuration to require both of them, since the driver is provided to the DatabaseConfiguration object below.

Is there something you would suggest instead?